backend.native/tests/runtime/workers/lazy1.kt

```diff
@@ -9,14 +9,40 @@ import kotlin.test.*
 
 import kotlin.native.concurrent.*
 
-class Leak {
-    val leak by lazy { this }
+class SelfReference {
```

Comment:
Can a `lazy {}` leak now when global cyclic collector is disabled?

Reply:
Yes, as expected.